osf_tests/factories.py

```diff
@@ -746,6 +760,7 @@ def _create(cls, target_class, *args, **kwargs):
         user = kwargs.pop('creator', None) or instance.creator
         instance.save()
 
+        GuidVersionsThroughFactory(referent=instance,)
```

Comment:
As mentioned above and in today's meeting, we can create these relationship in `_create()`